lib/src/platform_text_field.dart

```diff
@@ -53,6 +53,54 @@ const BoxDecoration _kDefaultRoundedBorderDecoration = BoxDecoration(
   borderRadius: BorderRadius.all(Radius.circular(5.0)),
 );
 
+InputDecoration _inputDecorationWithHint(String hint, InputDecoration inputDecoration) {
+  return InputDecoration(
+    alignLabelWithHint: inputDecoration?.alignLabelWithHint ?? null,
+    border: inputDecoration?.border ?? null,
+    contentPadding: inputDecoration?.contentPadding ?? null,
+    counter: inputDecoration?.counter ?? null,
+    counterStyle: inputDecoration?.counterStyle ?? null,
+    counterText: inputDecoration?.counterText ?? null,
+    disabledBorder: inputDecoration?.disabledBorder ?? null,
+    enabled: inputDecoration?.enabled ?? null,
```

Comment:
```suggestion
    enabled: inputDecoration?.enabled ?? true,
```
Matches the InputDecoration of setting a variable in the constructor